package-lock.json

```diff
@@ -7759,7 +7759,7 @@
                 "@babel/runtime": "^7.3.1",
                 "highlight.js": "~9.13.0",
                 "lowlight": "~1.11.0",
-                "prismjs": "^1.21.0",
+                "prismjs": "^1.8.4",
```

Comment:
Is this OK / what we would expect?

Reply:
yeah, now the we don't run `npx npm-force-resolutions` when we run `npm ci`, the corrections made to the `package-lock.json` are now being retained. `npm-force-resolutions` was previously modifiying the constraints and `npm ci` didn't set it back again. Now `npm ci` doesn't run `npm-force-resolutions` there's nothing to correct. 🤯

Reply:
🤯 ⚡